### ISSUE TITLE
Comments of types are deleted

### ISSUE THREAD

**Issue by yostools** [nicolasdao/graphql-s2s#17]
Hello Niclas, thank you very much for your great extension.

During the integration into my project, I noticed that comments fields are retained, but the comments of types are lost. Regardless of whether it is a single-line comment ("...") or a multiline comment ("""...""").

In the following example, the comment on the "Comparison operators" is lost and the comment on the "EQ" is retained:

```
# ==================================================================================================================
# Enums
# ==================================================================================================================

"""
Comparison operators
See [Comparison Query Operators](https://docs.mongodb.com/manual/reference/operator/query-comparison/)
"""
enum ComparisonOperator {
 
  """
  Specifies equality condition. The [$eq](https://docs.mongodb.com/manual/reference/operator/query/eq/) operator 
  matches documents where the value of a field equals the specified value.
  """
  EQ
}
```
The comments introduced with "#" disappear, which is the intention.

**Comment by nicolasdao** [comment]
Very interesting @yostools. I didn't even know the """ ... """ convention. 

I'm on holidays now. I'll integrate that ASAP.

**Comment by nicolasdao** [comment]
Closed with fix #32.
